src/main/scala/org/scalacheck/ScalaCheckFramework.scala

```diff
@@ -39,7 +39,8 @@ private abstract class ScalaCheckRunner extends Runner {
   def serializeTask(task: Task, serializer: TaskDef => String) =
     serializer(task.taskDef)
 
-  def tasks(taskDefs: Array[TaskDef]): Array[Task] = {
+  def tasks(_taskDefs: Array[TaskDef | Null] | Null): Array[Task | Null] = {
```

Comment:
A Scala method overrides some Java method. We need to add |Null to the method's parameter types to make the override still work.